xdp-loader/README.org

```diff
@@ -114,6 +115,20 @@ Enable debug logging. Specify twice for even more verbosity.
 ** -h, --help
 Display a summary of the available options
 
+* The CLEAN command
```

Comment:
For consistency with the other commands, please add an explicit "the syntax for the =clean= command is" bit that includes the [ifindex] parameter.